package.json

```diff
@@ -5,16 +5,18 @@
   "homepage": "http://passmarked.com",
   "preferGlobal": true,
   "engines": {
-    "node": ">=0.12.0"
+    "node": ">=0.12"
   },
   "bin": {
     "passmarked": "index"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "coverage": "istanbul cover _mocha --report lcovonly -- test -R spec"
   },
   "author": "Passmarked <info@passmarked.com>",
   "contributors": [
```

Comment:
we don't have to have a contributors list, it's just customary. i only updated it because i was in "housekeeping" mode.
git keeps a list of contributors for us, so we can directly link to [contributors](https://github.com/passmarked/passmarked/graphs/contributors) if we want?